### ISSUE TITLE
export to excel

### ISSUE THREAD

**Issue by Ombry** [moneymanagerex/moneymanagerex#4917]
MMEX version:
 - [ ] 1.5.14
 - [ ] 1.5.15
 - [x] 1.5.18
 - [ ] 1.5.17 Beta
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [x] Windows 64
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

When I export a report to Excel, excel starts but the sheet is empty

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

1. 
2. 
3. 

…

Expected result:

Actual result:

# Additional information

If you have a backtrace for a crash or a warning, paste it here.

**Comment by tactilis** [comment]
@Ombry
Duplicate of https://github.com/moneymanagerex/moneymanagerex/issues/3861, which also includes a way to work around the problem.

**Comment by whalley** [comment]
Closing duplicate
